stubs/openpyxl/openpyxl/drawing/image.pyi

```diff
@@ -1,9 +1,15 @@
-from _typeshed import Incomplete
+from types import ModuleType
+from typing_extensions import Literal
+
+from PIL import Image as ImageModule
+
+# Using "object" because modules can't by used as type
```

Comment:
There's a typo here, but I'm also not sure I really understand this comment

```suggestion
# Using "object" because modules can't be used as types
```

Reply:
outdated comment. I have since then learned of `ModuleType` which is enough for the needs here.
```suggestion
```